DTS-STN/Service-Canada-Labs
# Font Optimization

# [Investigate font optimizations](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities?workitem=204266)

This PR includes changes to how we load in fonts. You can read about what these changes do [here](https://nextjs.org/docs/pages/building-your-application/optimizing/fonts)

## Test Instructions

1. yarn build + yarn start
2. See that fonts render across the site, including the HelpIcon modal on project pages
